### DISCUSSION TITLE
Access to Eleven Turbo v2

### DISCUSSION THREAD

**Discussion by jeffries7** [RageAgainstThePixel/com.rest.elevenlabs#57] [Q&A] [answered]
I can see that you can access Turbo v2 via the dashboard but is it possible to access this via code?
I assume setting this via the dashboard doesn't configure it for for getting voices via code

**Comment by StephenHodgson** [comment, marked as answer]
use `model` parameter in your request:
```csharp
var voiceClip = await api.TextToSpeechEndpoint.TextToSpeechAsync(message, voice, model: Model.EnglishTurboV2, cancellationToken: lifetimeCancellationTokenSource.Token);
```
